TYPO3.TypoScript/Classes/TYPO3/TypoScript/Core/Cache/ContentCache.php

```diff
@@ -237,11 +243,14 @@ public function getCachedSegment($uncachedCommandCallback, $typoScriptPath, $cac
     public function replaceCachePlaceholders(&$content, $addCacheSegmentMarkersToPlaceholders)
     {
         $cache = $this->cache;
+        $debugger = $this->debugger;
```

Comment:
ach so, ja, sehe ich jetzt den preg_replace_callback(), habe ich übersehen. Aber trotzdem, ich dachte, dass PHP $this implizit captured, mit "use" muss man lediglich lokale Variablen angeben. Aber vielleicht täusche ich mich da, muss ich mal selber ausprobieren. Wenn ich die Doku richtig interpretiere, soll genau das ab 5.4 gehen, also auf $this zugreifen ohne dass man use ($this) machen muss.

see http://php.net/manual/en/functions.anonymous.php